### PR TITLE
fix-bugs

### PR DESCRIPTION
**Pull request by RobertDebicki**

### CODE REVIEW

**Review by NerdbordMentor — APPROVED** [review]
## Code Review Report

**Task Goal:**  
Update **readme.md** file, and correct the wrong sentence

**Requirements:**  
1. Typo should be fixed in **readme.md** file

**Code Changes Review:**

The code changes provided are in line with the task requirements. The typo in the sentence `I do nto sea eny typo!` in the **readme.md** file has been corrected to `I do not see any typo!`. This change meets the requirement of fixing the typo in the **readme.md** file.

In conclusion, the provided code changes cover the task requirements.
